src/main/java/com/fasterxml/jackson/core/io/IOContext.java

```diff
@@ -115,19 +118,37 @@
      *
      * @since 2.13
      */
-    public IOContext(BufferRecycler br, ContentReference contentRef, boolean managedResource)
+    public IOContext(StreamReadConstraints streamReadConstraints, BufferRecycler br,
+                     ContentReference contentRef, boolean managedResource)
     {
+        _streamReadConstraints = streamReadConstraints == null ?
+                StreamReadConstraints.builder().build() :
+                streamReadConstraints;
         _bufferRecycler = br;
         _contentReference = contentRef;
         _sourceRef = contentRef.getRawContent();
         _managedResource = managedResource;
     }
 
+    @Deprecated // since 2.14
+    public IOContext(BufferRecycler br, ContentReference contentRef, boolean managedResource)
+    {
+        this(null, br, contentRef, managedResource);
+    }
+
     @Deprecated // since 2.13
     public IOContext(BufferRecycler br, Object rawContent, boolean managedResource) {
         this(br, ContentReference.rawReference(rawContent), managedResource);
     }
 
+    /**
+     * @return configuration for streaming reads
+     * @since 2.15
+     */
+    public StreamReadConstraints getStreamReadConfig() {
```

Comment:
let's change to `streamReadConstraints()`.

Reply:
done